vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/NumberField.java

```diff
@@ -234,4 +240,32 @@ public String getPattern() {
         return getPatternString();
     }
 
+    private static class Formatter implements SerializableFunction<Double, String> {
```

Comment:
My formatter made a lot of changes to this file, please check the line-length

Reply:
I reformatted  the whole file. See if helped

Reply:
There's still one difference in the import order: com.vaadin should go after java imports. But that's not so crucial so I don't mind.